litellm/llms/vllm/completion/handler.py

```diff
@@ -142,7 +159,7 @@ def batch_completions(
     )
     """
     try:
-        llm, SamplingParams = validate_environment(model=model)
+        llm, SamplingParams, optional_params = validate_environment(model=model, optional_params=optional_params)
```

Comment:
can you add a mock test w/ screenshot of this working?

similar to this - https://github.com/BerriAI/litellm/blob/246e3bafc89e19297fa42aabbfca6d25058e5989/tests/llm_translation/test_azure_ai.py#L48


ideally this test would **not** add the vllm sdk as a dep on the ci/cd pipeline (so maybe use a magicmock object here)